php/libraries/PageElementsFormatter.class.inc

```diff
@@ -0,0 +1,91 @@
+<?php
+
+class PageElementsFormatter {
```

Comment:
Helper class to render JSON elements based on documentation. Will remove the need to have smarty do it for you